isteven-multi-select.js

```diff
@@ -624,7 +624,7 @@ angular.module( 'isteven-multi-select', ['ng', 'infinite-scroll'] ).directive( '
                 if ( type.toUpperCase() === 'BUTTONLABEL' ) {
                     return label;
                 }
-                return $sce.trustAsHtml( label );
+                return label;
```

Comment:
Are you sure there are no valid uses of html here?  The one potential use I can think of is in the customer app environment selector, when it shows the lock icon when you can't change options.  Not sure if that's part of the label though or something else.

Reply:
It's just the label. The lock is actually part of the [overridden template in customer app](https://github.com/ApplauseAQI/customer-app/blob/fusion/public/src/template-overrides/isteven-multi-select.htm#L96).